src/main/java/com/std/tothebook/api/repository/MyBookCustomRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.std.tothebook.api.repository;
+
+import com.std.tothebook.api.domain.dto.FindMyBookResponse;
+import com.std.tothebook.api.domain.dto.FindMyBooksResponse;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface MyBookCustomRepository {
+
+    List<FindMyBooksResponse> findMyBookByUserId(long userId);
```

Comment:
```suggestion
    List<FindMyBooksResponse> findMyBooksByUserId(long userId);
```
리스트로 조회하는 거니까 Books가 좋을 듯 합니다!